cirq-google/cirq_google/json_test_data/cirq.google.QuantumExecutable.json_inward

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
nit: Do we need to create the inward files for `QuantumExecutable`? For many other serializable objects which have circuit as a member, we've just updated their existing json files and didn't create new ones (eg: CalibrationLayer.json). Since we already have inward files for the circuit types themselves, I think we can safely ignore creating them for types that contain circuits as members but I don't have strong opinions. 

In general, it'd be nice to define and follow a consistent guidance for all types.

Reply:
Fair enough. I've reverted the inward files for cirq_google.